src/core/Button/Button.tsx

```diff
@@ -59,6 +60,7 @@ const StyledButton = styled(
       {...passProps}
       className={classnames(className, {
         [`${baseClassName}--${variant}`]: variant !== 'default',
+        [fullWidthClassName]: fullWidth,
```

Comment:
This does not need array syntax as `classnames` can read the key-string. Array syntax is needed when using template-literals. Can be seen from the documentation https://github.com/JedWatson/classnames#usage

Reply:
Actually I was wrong, it needs the Computed property name
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names
as the wanted className is set to the constant not the constant key itself.